components/monitoring/README.md

```diff
@@ -0,0 +1,113 @@
+###On boarding to services to Monitoring
+
+#1. Metrics exporter services
+a. The intended service should export the metrics from the application so that prometheus is able to understand it. 
+b. For refernce https://prometheus.io/docs/instrumenting/writing_exporters/
+c. Exported port, service, route should be accessible to prometheus service.
+[Here](https://github.com/redhat-appstudio/service-provider-integration-operator/blob/main/config/rbac/auth_proxy_service.yaml) is example for the spi-system
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: controller-manager-metrics-service
+  namespace: system
+spec:
+  ports:
+  - name: metrics
+    port: 8443
+    protocol: TCP
+    targetPort: https
+  selector:
+    control-plane: controller-manager
+    app.kubernetes.io/name: service-provider-integration-operator
+```
+
+#2. Service monitors
+a.
+i. If service monitor is for prometheus it self
+* Add service monitor declaration for scrapping
```

Comment:
This is rendering strangely because the Markdown is not correct, but I think you could combine these two lines (29 and 30) and change scrapping -> scraping.